Controllers/Web/v1/UsersController.cs

```diff
@@ -66,6 +68,26 @@ public async Task<IActionResult> GetProfileByIdAsync([FromRoute]long id)
             return NotFound(CreateErrorResponse(StatusCodes.Status404NotFound.ToString(), "NotFound"));
 
         }
+
+        // I need your opinion on updating about for user :)
+        // Add it here? Or add it to every controller?
+        [HttpPatch("about")]
+        public async Task<IActionResult> UpdateAboutAsync([FromBody] UpdateUsersAboutDTO updateUsersAboutDTO)
+        {
+            if (!ModelState.IsValid)
+                return CustomBadRequest();
+
+            var identity = HttpContext.User.Identity as ClaimsIdentity;
+            long authenticatedUserId = authService.GetUserId(identity);
+
+            var user = await mainAppContext.Users.FindAsync(authenticatedUserId);
+            if (user == null)
+                return NotFound(CreateErrorResponse(StatusCodes.Status404NotFound.ToString(), "User not found"));
+
+            user.About = updateUsersAboutDTO.About;
+            await mainAppContext.SaveChangesAsync();
+
+            return Ok(CreateSuccessResponse(new { Message = "Users About section updated successfully" }));
```

Comment:
Please replace this line with `return Ok(CreateSuccessResponse("Users About section updated successfully"));`  as we are using this scheme for all similar responses.

Reply:
By mistake